python/graphscope/deploy/kubernetes/resource_builder.py

```diff
@@ -673,6 +696,9 @@ def add_engine_container(self, name, image, cpu, mem, **kwargs):
 class GSEtcdBuilder(DeploymentBuilder):
     """Builder for graphscope etcd."""
 
+    _requests_cpu = 1.0
```

Comment:
https://github.com/alibaba/GraphScope/blob/f807eef2a37aae7724670c688feb05a8302c9c29/python/graphscope/config.py#L50
So in default case, the etcd limits cpu will be **0.5**, which is lower than the default and nonchangeable **1.0**.
I'm not sure, seems that means there will be error if using all default parameter to create the session?